template/tsconfig.json

```diff
@@ -0,0 +1,34 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "lib": [
+      "esnext",
+      "esnext.asynciterable",
+      "dom"
+    ],
+    "esModuleInterop": true,
+    "allowJs": true,
+    "sourceMap": true,
+    "strict": true,
+    "noEmit": true,
+    "experimentalDecorators": true,
+    "baseUrl": ".",
+    "paths": {
+      "~/*": [
+        "./*"
+      ],
+      "@/*": [
+        "./*"
+      ]
+    },
```

Comment:
I think we need to follow the same logic as in the [jsconfig.json](https://github.com/nuxt/create-nuxt-app/blob/master/template/jsconfig.json) file:

```
    "paths": {
      <%_ if (server === 'adonis') { _%>
      "~/*": ["./resources/*"],
      "@/*": ["./resources/*"],
      <%_ } else { _%>
      "~/*": ["./*"],
      "@/*": ["./*"],
      <%_ } _%>
      "~~/*": ["./*"],
      "@@/*": ["./*"]
    }
```